pinot-clients/pinot-java-client/src/main/java/org/apache/pinot/client/JsonAsyncHttpPinotClientTransportFactory.java

```diff
@@ -60,4 +72,18 @@ public SSLContext getSslContext() {
   public void setSslContext(SSLContext sslContext) {
     _sslContext = sslContext;
   }
+
+  public JsonAsyncHttpPinotClientTransportFactory withConnectionProperties(Properties properties) {
+    _readTimeout = Integer.parseInt(properties.getProperty("brokerReadTimeout",
```

Comment:
Since we expect timeouts to be in milliseconds, we should add this to property name as well. e.g. `brokerReadTimeoutMs`